i18n/fr/code.json

```diff
@@ -863,5 +871,57 @@
   },
   "navbar.item.label.admin": {
     "message": "Admin"
+  },
+  "components.user.index.untis.account.button": {
+    "message": "Compte Untis"
+  },
```

Comment:
Use the translation cli, s.t. the "description" is automatically added too. It is more convenient too, since all the missing fields are added...

```bash
yarn write-translations --locale fr
```